lib/Logger.js

```diff
@@ -4,15 +4,21 @@ const LogLevel = require('./LogLevel.js');
 const RequestLogger = require('./RequestLogger.js');
 const unserializeUids = require('./Utils.js').unserializeUids;
 const Config = require('./Config.js');
+const DefaultFields = require('./DefaultFields.js');
```

Comment:
Well, all the code within werelogs adds the `.js`, so I don't plan on creating inconsistency with this PR. Can be a later cleanup :)